docs/source/deployment/airflow.md

```diff
@@ -0,0 +1,181 @@
+* [`kedro-airflow>=0.8`](https://github.com/kedro-org/kedro-plugins/tree/main/kedro-airflow) installed. We will use this plugin to convert the Kedro pipeline into an Airflow DAG.
+
+### Create, prepare and package example Kedro project
+
+In this section, we will create a new Kedro project equipped with an example pipeline designed to solve a typical data science task: predicting spaceflights prices. We will customise this project to ensure compatibility with Airflow, which includes enriching the Kedro `DataCatalog` with datasets previously stored only in memory and simplifying logging through custom settings. Following these modifications, we'll package the project for installation in an Airflow Docker container and generate an Airflow DAG that mirrors our Kedro pipeline.
+
+1. To create a new Kedro project, select the `example=yes` option to include example code. Additionally, to implement custom logging, select `tools=log`. Proceed with the default project name, but feel free to add any other tools as desired:
+
+    ```shell
+    kedro new --example=yes --name=new-kedro-project --tools=log
+    ```
+
+2. Navigate to your project's directory, create a new `conf/airflow` directory for Airflow-specific configurations, and copy the `catalog.yml` file from `conf/base` to `conf/airflow`. This setup allows you to customise the `DataCatalog` for use with Airflow:
+
+    ```shell
+    cd new-kedro-project
+    mkdir conf/airflow
+    cp conf/base/catalog.yml conf/airflow/catalog.yml
+    ```
+
+3. Open `conf/airflow/catalog.yml` to see the list of datasets used in the project. Note that additional intermediate datasets (`X_train`, `X_test`, `y_train`, `y_test`) are stored only in memory. You can locate these in the pipeline description under `/src/new_kedro_project/pipelines/data_science/pipeline.py`. To ensure these datasets are preserved and accessible across different tasks in Airflow, we need to include them in our `DataCatalog`:
+
+```yaml
+X_train:
+  type: pandas.CSVDataset
+  filepath: data/02_intermediate/X_train.csv
+
+X_test:
+  type: pandas.CSVDataset
+  filepath: data/02_intermediate/X_test.csv
+
+y_train:
+  type: pandas.CSVDataset
+  filepath: data/02_intermediate/y_train.csv
+
+y_test:
+  type: pandas.CSVDataset
+  filepath: data/02_intermediate/y_test.csv
+```
+
+In the example here we assume that all Airflow tasks share one disk, but for distributed environments you would need to use non-local file paths.
+
+4. Open `conf/logging.yml` and modify the `root: handlers` section to `[console]` at the end of the file. By default, Kedro uses the [Rich library](https://rich.readthedocs.io/en/stable/index.html) to enhance log output with sophisticated formatting. However, some deployment systems, including Airflow, don't work well with Rich. Therefore, we're adjusting the logging to a simpler console version. For more information on logging in Kedro, you can refer to the [Kedro docs](https://docs.kedro.org/en/stable/logging/index.html).
```

Comment:
I was about to ask about https://github.com/kedro-org/kedro-plugins/issues/603 good to have this here 👍🏼